spec/faker/bot/commands/list_spec.rb

```diff
@@ -7,23 +7,43 @@
   let(:options) { {} }
   let(:command) { Faker::Bot::Commands::List.new(options) }
 
-  before do
-    command.execute(output: output)
-  end
-
   context 'when single `list` command' do
     it 'executes successfully' do
+      command.execute(output: output)
```

Comment:
I see you're warming up to the "mystery guest mitigation" style 😁

Reply:
I am! Maybe we should talk about standardizing on a spec style across the repo. If you want to standardize on the mystery guest mitigation style, we can remove the ‘let’s and have code that’s duplicated, but easier to reason about.

Reply:
I'm definitely open to that. I wonder if we can configure [rubocop-rspec](https://github.com/rubocop-hq/rubocop-rspec) to enforce those rules. 🤔 But I really don't have any strong feelings on this one.  🙁